WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -148,6 +157,17 @@ protected SendControlViewModel(Wallet wallet, string title)
 				}
 			});
 
+			this.WhenAnyValue(x => x.FeeDisplayFormat)
+				.ObserveOn(RxApp.TaskpoolScheduler)
+				.Subscribe(x =>
+				{
+					Global.UiConfig.FeeDisplayFormat = (int)x;
+
+					this.RaisePropertyChanged(nameof(TransactionFeeInfo));
+
+					Global.UiConfig.ToFile();
```

Comment:
What does this do?

Reply:
I'm not sure this is the best approach. Two lines above you raise property changed event so maybe it should be in a handler that handles this change. Anyway, I think this operation should be actually logged as in https://github.com/zkSNACKs/WalletWasabi/blob/4695109648428687643bc5ea3361aa7a459d5bb6/WalletWasabi.Gui/MainWindow.xaml.cs#L115.